CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 #### 2.0.4 (Unreleased)
 
+- [#482](https://github.com/influxdata/clockface/pull/482): [Breaking] Remove render props from `ResourceCard` in favor of `children`
```

Comment:
>[Breaking]

this implies that this should be 3.0.0, right? if we're breaking the interface, it's a major breaking change, which is a major bump in semver. What do you think?